### DISCUSSION TITLE
Initial CSS load clears background-image from <select> conflicting with tailwindcss/forms styling

### DISCUSSION THREAD

**Discussion by smallsam** [jrief/django-formset#157] [General]
I'm using [tailwindcss/forms base ](https://github.com/tailwindlabs/tailwindcss-forms) and this re-styles the dropdown arrow using a background-image on the select box: https://github.com/tailwindlabs/tailwindcss-forms/blob/master/src/index.js#L152

However when loading formset, the js loads a background-image: none !important style into the <head><style>,  [presumably sourced from](https://github.com/jrief/django-formset/blob/964faf74e9a9ef9ad58522bf2bdd21f78c61aff8/client/django-formset/DjangoFormset.scss#L153) strangely once loading of formset js is complete this style has been removed, but for some reason Chrome/Chromium at least doesn't re-render the page to include the background-image arrow.

Honestly I'm not sure of the best way to tackle this issue and would welcome any thoughts?

**Comment by jrief** [comment]
could you please share a screenshot, or better a screencast?
